client/src/pages/CreateWorkout/components/AddExercise.jsx

```diff
@@ -1,59 +1,60 @@
-import React from "react";
+import React, { useEffect } from "react";
 import ExerciseCard from "@client/src/components/ExerciseCard/ExerciseCard";
 import { useState } from "react";
 
-const AddExercise = () => {
+const AddExercise = ({ handleExerciseSubmitted }) => {
   const [exercises, setExercises] = useState([]);
   const [exercise, setExercise] = useState({
     name: "",
-    // duration: { minutes: "", seconds: "" },
-    minutes: "",
-    seconds: "",
+    durationMinutes: "",
+    durationSeconds: "",
     weight: "",
     reps: "",
     sets: "",
   });
 
   const handleChange = (e) => {
-    // if (e.target.name === "minutes" || e.target.name === "seconds") {
-    //   setExercise({
-    //     ...exercise,
-    //     duration: { ...exercise.duration, [e.target.name]: e.target.value },
-    //   });
-    // } else {
-    //   setExercise({ ...exercise, [e.target.name]: e.target.value });
-    // }
     setExercise({ ...exercise, [e.target.name]: e.target.value });
   };
 
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setExercises([...exercises, exercise]);
```

Comment:
ah I see where you're using the state now from my prior PR comment.

Same comment as other PR - this could be `setExercises(prevExercises -> ([...prevExercises, exercise]))`